treeherder/model/error_summary.py

```diff
@@ -166,7 +166,13 @@ def is_helpful_search_term(search_term):
 
     ]
 
-    return len(search_term) > 4 and not (search_term in blacklist)
+    shortname_whitelist = [
+        'damp',
+        'tps'
+    ]
+
+    return (len(search_term) > 4 and not (search_term in blacklist)) or \
+           len(search_term) <= 4 and search_term in shortname_whitelist
```

Comment:
I think this might be clearer as:
```python
if search_term in blacklist:
    return False

if search_term in whitelist:
    return True

return len(search_term) > 4
```